### PR TITLE
Fixed Broken Link for Metasploit Goliath in GSOC'23 page.

### PR DESCRIPTION
**Pull request by samsepi0x0**
The link for the Metasploit-Data-Service-Enhancements-Goliath was not working on the website as no such route existed.

image
 
Added the github docs link mentioning Goliath.

### CODE REVIEW

**Comment by gwillcox-r7** [comment]
You linked to the blob on GitHub not the actual page on docs.metasploit.com. Furthermore we should be trying to use relative links vs hardcoding full URLs. See https://github.com/rapid7/metasploit-framework/tree/master/docs#adding-links for more info as well as https://github.com/rapid7/metasploit-framework/blob/master/docs/navigation.rb#L926-L928 and https://github.com/rapid7/metasploit-framework/blob/master/docs/build.rb#L220-L227

**Comment by samsepi0x0** [comment]
> You linked to the blob on GitHub not the actual page on docs.metasploit.com. Furthermore we should be trying to use relative links vs hardcoding full URLs. See https://github.com/rapid7/metasploit-framework/tree/master/docs#adding-links for more info as well as https://github.com/rapid7/metasploit-framework/blob/master/docs/navigation.rb#L926-L928 and https://github.com/rapid7/metasploit-framework/blob/master/docs/build.rb#L220-L227

I was wondering that since I found documentation on both blob and docs.metasploit.com. Thanks to the links above I was able to generate the docs locally before committing here.

File: docs/metasploit-framework.wiki/GSoC-2023-Project-Ideas.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Difficulty: 4/5
 
 Enhance existing Metasploit Goliath dashboard that allows observation of an active engagement. Data visualization would include, but not be limited to: host node graph with activity indicators and heat maps. The main idea here is to create a visualization tool that helps users understand data that has been gathered into Metasploit during usage in some useful way. Proposals should note where the service will live, how a user will use the service, and how you will provide a maintainable and extendable consumer for the data that is exposed.
 
-See [Metasploit 'Goliath' Demo (msf-red)](https://www.youtube.com/watch?v=hvuy6A-ie1g&feature=youtu.be&t=176) for a demo video of Goliath in action. You can also read more on Metasploit Goliath at [Metasploit-Data-Service-Enhancements-(Goliath)](./Metasploit-Data-Service-Enhancements-Goliath)
+See [Metasploit 'Goliath' Demo (msf-red)](https://www.youtube.com/watch?v=hvuy6A-ie1g&feature=youtu.be&t=176) for a demo video of Goliath in action. You can also read more on Metasploit Goliath at [Metasploit-Data-Service-Enhancements-(Goliath)](/docs/development/roadmap/metasploit-data-service-enhancements-goliath.html)
```
Comment (adfoster-r7) [inline docs/metasploit-framework.wiki/GSoC-2023-Project-Ideas.md:54]:
```suggestion
See [Metasploit 'Goliath' Demo (msf-red)](https://www.youtube.com/watch?v=hvuy6A-ie1g&feature=youtu.be&t=176) for a demo video of Goliath in action. You can also read more on Metasploit Goliath at [[Metasploit-Data-Service-Enhancements-(Goliath)|./Metasploit-Data-Service-Enhancements-Goliath]]
```

The docs site is currently using the Github wiki flavored markdown syntax for linking to other docs pages

Comment (samsepi0x0) [reply docs/metasploit-framework.wiki/GSoC-2023-Project-Ideas.md:54]: I was not aware of this, I'll check and change accordingly.

Comment (adfoster-r7) [reply docs/metasploit-framework.wiki/GSoC-2023-Project-Ideas.md]: Thanks! :+1:

**Comment by adfoster-r7** [comment]
## Release Notes

Fixes a broken link within Metasploit's Google Summer of Code 2023 Project Ideas

**Pull request merged**